src/components/TodoItem.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./TodoItem.css";
+class TodoItem extends React.Component {
```

Comment:
This will cause a performance issue, resulting in rerendering all items after every time the new item is added/deleted.
Use `PureComponent` instead and read about that and `shouldComponentUpdate()` lifecycle method.